main.py

```diff
@@ -2,17 +2,29 @@
 
 This NApp implements Oplenflow multi tables
 """
-# pylint: disable=unused-argument
+# pylint: disable=unused-argument, too-many-arguments, too-many-public-methods
+# pylint: disable=attribute-defined-outside-init
 import pathlib
+import time
+from typing import Dict, Optional
 
+import httpx
+import tenacity
+from httpx._exceptions import RequestError
```

Comment:
```suggestion
from httpx import RequestError
```

You can import it directly from `httpx` instead of a "private" module

Reply:
Fixed